.github/workflows/build.yml

```diff
@@ -38,10 +35,25 @@ jobs:
         with:
           java-version: "adopt@1.8"
 
-      - name: Build and run tests
-        run: sbt "project agent" ++${{matrix.scala}} test doc "project model" ++${{matrix.scala}} test doc
+      - name: Build and run unit tests
+        run: sbt "project agent" test doc "project model" test doc
 
-  build-server:
+  build-server-unit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: coursier/cache-action@v5
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v14
+        with:
+          java-version: "adopt@1.11.0-11"
+
+      - name: Build and run unit tests
+        run: sbt "project server" test
```

Comment:
no `doc` ?

Reply:
This was called once before my change. I kept it in the last call.